### PR TITLE
change the option

### PR DESCRIPTION
**Pull request by SinceJune**

### CODE REVIEW

**Comment by SinceJune** [comment]
the readme in root is invoking inside the docker images, i think it should be replaced.

i choose the `tables` as expand name of `-n`, because if using `table-names`, when specify it in `dtail.yml`, it must be `table_names: nginx_json`, it was confused

**Comment by PureWhiteWu** [comment]
I don't think this is okay, since `-n` seems confused for `table`

File: tests/unittests/tools/test_dtail.py [hunk 1]
```diff
@@ -9,15 +9,20 @@
...
         raise Exception
     req.append('-a')
     req.append(addr)
 
-    if num:
+    if table_names:
         req.append('-n')
+        req.append(table_names)
+
+    if num:
```
Comment (javasoze) [inline tests/unittests/tools/test_dtail.py]: let's call it count for c.

File: tools/dtail.py [hunk 2]
```diff
@@ -13,8 +13,11 @@
 CONFIG_PATH = "~/.dashbase/dtail.yml"
 
 parser = argparse.ArgumentParser(description='dtail for Dashbase')
-parser.add_argument("-n", "--num", type=int, default=10, help="number of lines to return")
-parser.add_argument("-a", "--addr", type=str, default=None, help="Log address, e.g. tables@host:port, for multiple tables, delimit by comma")
+parser.add_argument("-c", "--count", type=int, default=10, help="number of lines to return")
+parser.add_argument("-n", "--tables", type=str, default=None,
```
Comment (javasoze) [inline tools/dtail.py]: -n, --names

File: tools/dtail.py [hunk 3]
```diff
@@ -45,16 +48,17 @@ def main():
         print("or you can edit config file at: {}".format(CONFIG_PATH))
         print("example: ")
         print("addr: nginx_json@staging.dashbase.io:9876")
```
Comment (javasoze) [inline tools/dtail.py]: addr should only be the host name

**Comment by javasoze** [comment]
@PureWhiteWu -n => --names

**Comment by SinceJune** [comment]
update `tables` -> `names`

**Pull request merged**